### PR TITLE
Release notes for 2022-09-10

### PR DESCRIPTION
**Pull request by justindavies**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-09-10
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+* Asia East has now been changed to the 2nd release region. New release changes will reach to Asia East after US West Central, and before UK South.   Follow this via [AKS-Release-Tracker](https://releases.aks.azure.com/).
```
Comment (kaarthis) [inline CHANGELOG.md:8]: This can be removed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+* To avoid disruptions stemming from unmanaged Canonical nightly security updates, AKS will disable unmanaged Canonical nightly updates by 2 September 2023, on clusters that haven’t specified an update option explicitly, mapping to the option `None` in the [node OS upgrade](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image) channel feature. AKS strongly recommends proactively moving to [auto-upgrade node-image](https://learn.microsoft.com/azure/aks/auto-upgrade-cluster) or [node OS upgrade channel - SecurityPatch or NodeImage options](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image); you can set [maintenance windows](https://learn.microsoft.com/azure/aks/planned-maintenance) for these channels.
+
+### Release notes 
+* Features
+  * Enable [HonorPVReclaimPolicy](https://kubernetes.io/blog/2021/12/15/kubernetes-1-23-prevent-persistentvolume-leaks-when-deleting-out-of-order/ ) for Azure Disk CSI driver 1.28
```
Comment (kaarthis) [inline CHANGELOG.md:14]: would it make sense to elaborate a bit on this with what this is about eg: The Reclaim Policy is used to determine the actions that need to be taken by the storage backend on deletion of the PV. Where the reclaim policy is Delete, the expectation is that the storage backend releases the storage resource that was allocated for the PV. In essence, the reclaim policy needs to honored on PV deletion.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
+ 
+* Component Updates
+  * Upgrade Azure Disk CSI version to [v1.28.3](https://github.com/kubernetes-sigs/azuredisk-csi-driver/releases/tag/v1.28.3) on K8S 1.27
```
Comment (kaarthis) [inline CHANGELOG.md:17]: Nit: Updated instead of upgrade. Missing a full stop at the end.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,33 @@
...
+  * Upgrade Azure File CSI version to [v1.28.3](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.28.3) on K8S 1.27, [v1.26.6](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.26.6) on K8S 1.26, [v1.24.7](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.24.7) on K8S 1.25
```
Comment (kaarthis) [inline CHANGELOG.md:18]: same as above.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,33 @@
...
+  * AKS Ubuntu 18.04 image has been updated to [AKSUbuntu-1804-202309.06.0](vhd-notes/aks-ubuntu/AKSUbuntu-1804/202309.06.0.txt).
+  * AKS Ubuntu 22.04 image has been updated to [AKSUbuntu-2204-202309.06.0](vhd-notes/aks-ubuntu/AKSUbuntu-2204/202309.06.0.txt).
+  * Azure Linux image has been updated to [AzureLinux-202309.06.0](vhd-notes/AzureLinux/202309.06.0.txt).
+
+* Behavioral changes
+  * Update admissions enforcer to ignore "kubernetes.azure.com/managedby" and "control-plane" namespaces to fix [this issue](https://github.com/Azure/AKS/issues/1771).
+  * "kubernetes.azure.com/managedby" label added to aks managed namespaces (kube-system, gatekeeper-system, tigera-system, calico-system)
```
Comment (kaarthis) [inline CHANGELOG.md:25]: is there a doc update or PR (on doc ) for this?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,33 @@
...
+  * Stopped nodepools will be upgraded during an Auto Upgrade operation.  The upgrade will apply to nodes when the nodepool is started.
```
Comment (kaarthis) [inline CHANGELOG.md:26]: Let me add a doc PR to this today.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,33 @@
...
+  * Added priorityClassName system-node-critical property to all KEDA add-on pods to fix [this issue](https://github.com/Azure/AKS/issues/3780).
+  * Update Konnectivity agent to rotate certificate if CA is not expired but the certificate has expired.
+  * We will now check that your cluster has less than 400 nodes when an upgrade operation is requests and using Kubenet (400 being the node limit for Kubenet).
```
Comment (kaarthis) [inline CHANGELOG.md:29]: is requested* ?

File: CHANGELOG.md (hunk 5 above)
Comment (justindavies) [reply CHANGELOG.md:25]: There isn't it's a fix on what happens during upgrades on stopped nodepools

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,31 @@
...
+* To avoid disruptions stemming from unmanaged Canonical nightly security updates, AKS will disable unmanaged Canonical nightly updates by 2 September 2023, on clusters that haven’t specified an update option explicitly, mapping to the option `None` in the [node OS upgrade](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image) channel feature. AKS strongly recommends proactively moving to [auto-upgrade node-image](https://learn.microsoft.com/azure/aks/auto-upgrade-cluster) or [node OS upgrade channel - SecurityPatch or NodeImage options](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image); you can set [maintenance windows](https://learn.microsoft.com/azure/aks/planned-maintenance) for these channels.
```
Comment (bridgetkromhout) [inline CHANGELOG.md:9]: >  AKS will disable unmanaged Canonical nightly updates by 2 September 2023

This is a time in the past - so either the date is confusing or "will" is confusing.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,31 @@
...
+
+### Release notes 
+* Features
+  * Enable [HonorPVReclaimPolicy](https://kubernetes.io/blog/2021/12/15/kubernetes-1-23-prevent-persistentvolume-leaks-when-deleting-out-of-order/) for Azure Disk CSI driver 1.28, fixing an issue where in some Bound Persistent Volume (PV) – Persistent Volume Claim (PVC) pairs, the ordering of PV-PVC deletion determines whether the PV delete reclaim policy is honored.
+ 
+* Component Updates
+  * Updated Azure Disk CSI version to [v1.28.3](https://github.com/kubernetes-sigs/azuredisk-csi-driver/releases/tag/v1.28.3) on K8S 1.27.
+  * Updated Azure File CSI version to [v1.28.3](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.28.3) on K8S 1.27, [v1.26.6](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.26.6) on K8S 1.26, [v1.24.7](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.24.7) on K8S 1.25.
```
Comment (bridgetkromhout) [inline CHANGELOG.md:17]: We have some v1.24.x info listed here for 1.25 - is this outdated from a previous version?

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,31 @@
...
+  * AKS Ubuntu 18.04 image has been updated to [AKSUbuntu-1804-202309.06.0](vhd-notes/aks-ubuntu/AKSUbuntu-1804/202309.06.0.txt).
+  * AKS Ubuntu 22.04 image has been updated to [AKSUbuntu-2204-202309.06.0](vhd-notes/aks-ubuntu/AKSUbuntu-2204/202309.06.0.txt).
+  * Azure Linux image has been updated to [AzureLinux-202309.06.0](vhd-notes/AzureLinux/202309.06.0.txt).
+
+* Behavioral changes
```
Comment (palma21) [inline CHANGELOG.md:22]: Suggest having this before component updates, per the usual order

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,31 @@
...
+* To avoid disruptions stemming from unmanaged Canonical nightly security updates, AKS will disable unmanaged Canonical nightly updates by 2 September 2023, on clusters that haven’t specified an update option explicitly, mapping to the option `None` in the [node OS upgrade](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image) channel feature. AKS strongly recommends proactively moving to [auto-upgrade node-image](https://learn.microsoft.com/azure/aks/auto-upgrade-cluster) or [node OS upgrade channel - SecurityPatch or NodeImage options](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image); you can set [maintenance windows](https://learn.microsoft.com/azure/aks/planned-maintenance) for these channels.
+
+### Release notes 
+* Features
+  * Enable [HonorPVReclaimPolicy](https://kubernetes.io/blog/2021/12/15/kubernetes-1-23-prevent-persistentvolume-leaks-when-deleting-out-of-order/) for Azure Disk CSI driver 1.28, fixing an issue where in some Bound Persistent Volume (PV) – Persistent Volume Claim (PVC) pairs, the ordering of PV-PVC deletion determines whether the PV delete reclaim policy is honored.
```
Comment (palma21) [inline CHANGELOG.md:13]: This is a bugfix?

**Review by palma21 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 11 above)
Comment (justindavies) [reply CHANGELOG.md:13]: Wellll, it is technically but is using a feature gate.  I'll move it

**Review by palma21 — APPROVED** [review]

**Review by kaarthis — APPROVED** [review]

**Pull request merged**